tracker/table.go

```diff
@@ -149,8 +156,8 @@ func (v Table) SetCursorY(y int) {
 
 // Order methods
 
-func (v *Order) Table() Table {
```

Comment:
This seems unrelated to the main commit, please use `style:` commits if you think there's really need to harmonize receiver names.

Reply:
turns out that I accidentally triggered a GoLand recommendation. That's just polluting this pull request. Undid that.